main.js

```diff
@@ -25,3 +56,93 @@ overlayLinkTwo.addEventListener('click', () => {
 overlayLinkThree.addEventListener('click', () => {
   overlay.style.display = 'none';
 });
+
+// Portfolio projects in an array
+const projects = [
+  {
+    id: 1,
+    name: 'Tonic',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    featured_img: './images/desktop-works/work-one.png',
+    tecnologies: ['html', 'css', 'javascript'],
+    live_link: 'https://github.com/OyePriscilla/MyPortfolioSetup',
```

Comment:
- Please on your mobile view the popup functionality does not work, please check it out and fix it.

If you find this feedback confusing, please do reach out to me on slack `@kanu mike chibundu`